pkg/kubelet/container/runtime.go

```diff
@@ -309,6 +310,8 @@ type Status struct {
 	ID ContainerID
 	// Name of the container.
 	Name string
+	// ID of the sandbox to which this container belongs.
+	PodSandboxId string
```

Comment:
nit: ID must be both uppercased:

```suggestion
	PodSandboxID string
```